docs/nx/playwright/visualRegression.md

```diff
@@ -0,0 +1,67 @@
+### @ensono-stacks/playwright:visualRegression
+
+<details>
+<summary>Adds native or cloud based visual regression to your test suite</summary>
+
+The _visualRegression_ generator provides you with the option to scaffold visual regression tests and configuration through a cloud based provider or Playwrights native visual comparison API.
+
+## Usage
+
+```bash
+nx g @ensono-stacks/playwright:visualRegression
+```
+
+Upon calling the _visualRegression_ generator you will be presented with a number of options:
+- What type of visual regression tests would you like to use?
+    - native: Generate visual regression tests using Playwrights native [visual comparison api](https://playwright.dev/docs/test-snapshots)
+    - applitools: Generate visual regression tests using the [`@applitools/eyes-playwright`](https://www.npmjs.com/package/@applitools/eyes-playwright) plugin and scaffold an example visual regression test batch
+
+### Command line arguments
+
+The following command line arguments are available:
+
+| Option                | Description                                                       | Type      | Accepted Values                           |Default    |
+| ---------------       | --------------------------------------------------------------    | ---       | ---                                       | ---       |
+| --project             | The name of the existing playwright test app to enhance       | string   |                              |           |
+| --visualRegression, -v   | Method used to conduct visual testing                            | string    | [choices: "native", "applitools"] | none      |
+
+### Generator Output
+
+##### Playwright with native visual comparisons
+Opting to scaffold *native* visual testing will make a number of amendments to your test projects configuration:
+1. [playwright.config.ts snapshot configuration](../../testing/testing_in_nx/playwright_visual_testing#snapshot-configuration): Configuration for your visual tests
+2. [playwright-visual-regression.spec.ts](../../testing/testing_in_nx/playwright_visual_testing#sample-tests): Sample test showcasing how to perform visual testing using playwrights native [visual comparison api](https://playwright.dev/docs/test-snapshots).
+```text title="Generated files"
```

Comment:
project.json inside the e2e project also gets updated here for native
<img width="993" alt="image" src="https://user-images.githubusercontent.com/101654886/221879396-8ab379f6-c516-4a88-a0c3-a824923dc783.png">